### PR TITLE
add better alarm description

### PR DESCRIPTION
**Pull request by PetroSemeniuk**

### CODE REVIEW

File: library/aws/alarms.ts [hunk 1]
```diff
@@ -236,7 +236,7 @@ export function createEc2HighMemAlarm(
     dimensions: {
       InstanceId: ec2.id.value,
     },
-    alarm_description: 'Sustained high memory for application server',
```
Comment (PetroSemeniuk) [inline library/aws/alarms.ts:239]: @timbod7 @paul-thompson-helix I added only three lines above for better alarm name but this PR contains whole set which plp made over time.

You are ok with merging in one go?

Presumably they were already reviewed/agreed

Comment (timbod7) [reply library/aws/alarms.ts:239]: The commits:

* Add more r5 instances
* better description for alarm

both look fine. I have no idea about the other one.

Comment (timbod7) [reply library/aws/alarms.ts:239]: > Presumably they were already reviewed/agreed

They are not on master, so why do you presume they were reviewed?

File: library/camus2/releaseurl.ts [hunk 2]
```diff
@@ -1 +1 @@
-export const release_url: string = "https://github.com/helix-collective/camus2/releases/download/0.17/camus2.x86_64-linux.gz -O /opt/bin/camus2.gz";
+export const release_url: string = "https://github.com/helix-collective/camus2/releases/download/0.16/camus2.x86_64-linux.gz -O /opt/bin/camus2.gz";
```
Comment (PetroSemeniuk) [inline library/camus2/releaseurl.ts:1]: @paul-thompson-helix  @timbod7  @barryskal  @bacek   do you guys remember why would plp downgrade camus version?

Comment (timbod7) [reply library/camus2/releaseurl.ts:1]: This is a shared submodule, not plp code? So a revert here ultimately affects all users of this module when they update.

**Review by paul-thompson-helix — CHANGES_REQUESTED** [review]
Flagging with 'request changes' while I look further into it - 
I can't see why there should be a revert (or a double revert) of the upgrade c2...

**Comment by paul-thompson-helix** [comment]
@PetroSemeniuk plp is a couple of commits off to the side on hx-terraform.

To land in hx-terrform master it should be rebased and any plp-specific commits omitted.
The commit that reverts the upgraded c2 ADL-generated typescript (https://github.com/helix-collective/hx-terraform/pull/19/commits/45ed7fa3564d94abc9dbc5d51d11443f09f3e780) is not intended to land in hx-terraform master.

The commit "Add more r5 instances" is already in hx-terraform master.
https://github.com/helix-collective/hx-terraform/pull/19/commits/b2ce87c6925d1046211e03994f6c96a94df89b3d

I pushed a branch https://github.com/helix-collective/hx-terraform/compare/dev-pault-plp-commits if you want to note that and change this PR.
Otherwise can do a new PR of that branch (1 commit).

**Comment by paul-thompson-helix** [comment]
Replacing with a new PR for one commit cherry-picked for the 'better description for alarm'

**Pull request closed without merging**